### PR TITLE
commit for HOP LoKi Functor

### PR DESCRIPTION
**Pull request by gandreassi**

### CODE REVIEW

**Comment by alexpearce** [comment]
GitHub is telling me you added quite a few files 😅 

Woah!

Did you mean to add all these files? It looks like you added the `.svn` folder from somewhere.

**Comment by apuignav** [comment]
Now that the number of files is fixed, I have some general comments before getting into the gory details: I would update the structure of the note to accomodate the `TupleTool` team and to make it easier to read.

I would do the following:
- Title: Introducing the HOP mass in the LHCb software
- Introduction: you describe the HOP mass, reference the INT Note and give the goals (do a LoKi functor and a TupleTool).
- Physics details: you explain the gory physics details and how the tools treat all the possible cases of decays (B->Kemu, B->K*ee, with an intermediate Jpsi, etc).
- LoKi functor implementation
  - How to implement a LoKi functor?
- TupleTool implementation
  - How to implement a TupleTool
- Results and comparison between tools
- Conclusions

What do you think?

File: contrib-04-HOP-LoKi-Functor/ProceedingHOP.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+In order to solve this task we have to make changes to the DaVinci software package. So first, we will need to set up a development environment. Use the `lb-dev` command:
+
+`lb-dev --name DaVinciDevWithHOPLoKi DaVinci v39r1p1`
+
+This creates `DaVinciDevWithHOPLoKi` folder, which will be our working folder. 
+
+The LoKi toolkit lives in the `Phys/LoKiPhys` package. So first we move into the `DaVinciDevWithHOPLoKi` folder and download the package from the repository.
+
+```bash
+mv ./DaVinciDevWithHOPLoKi
```
Comment (apuignav) [inline contrib-04-HOP-LoKi-Functor/ProceedingHOP.md]: `cd` right?

Comment (gandreassi) [inline contrib-04-HOP-LoKi-Functor/ProceedingHOP.md]: Yes,thanks. I'm fixing this.

**Comment by cmarinbe** [comment]
I think we are ready to merge now. @gandreassi can you remove the "WIP" at the beginning of the pull request title? 
Of course, any comments are welcome :)

**Comment by apuignav** [comment]
Merged!

**Pull request merged**